exonum-java-binding/core/rust/exonum-java/src/node.rs

```diff
@@ -26,6 +26,10 @@ use java_bindings::{
     Command, Config, EjbCommand, EjbCommandResult, Executor, InternalConfig, JavaRuntimeProxy,
 };
 
+use java_bindings::exonum::{
```

Comment:
Not everyone likes nested uses, but since we've started doing so I'd suggest following this way 🙃